Makefile

```diff
@@ -172,14 +172,18 @@ soak-eunit: couch
 	while [ $$? -eq 0 ] ; do $(REBAR) -r eunit $(EUNIT_OPTS) ; done
 
 .PHONY: elixir
-elixir: elixir-check-formatted
+elixir: elixir-check-formatted elixir-credo
 	@rm -rf dev/lib
 	@dev/run -a adm:pass --no-eval test/elixir/run
 
 .PHONY: elixir-check-formatted
 elixir-check-formatted:
 	@cd test/elixir/ && mix format --check-formatted
 
+.PHONY: elixir-credo
```

Comment:
Can you add a comment in the two Makefiles that explains what `credo` actually is? For people unfamiliar with Elixir it may not be clear that this is a linting/formatting tool.